services/web/server/src/simcore_service_webserver/notifications/_rabbitmq_consumers.py

```diff
@@ -192,11 +193,24 @@ async def _events_message_parser(app: web.Application, data: bytes) -> bool:
 async def setup_rabbitmq_consumers(app: web.Application) -> AsyncIterator[None]:
     with log_context(_logger, logging.INFO, msg="Subscribing to rabbitmq channels"):
         rabbit_client: RabbitMQClient = get_rabbitmq_client(app)
-
-        for exchange_name, parser_fct, queue_kwargs in EXCHANGE_TO_PARSER_CONFIG:
-            await rabbit_client.subscribe(
-                exchange_name, functools.partial(parser_fct, app), **queue_kwargs
+        subscribed_queues = await logged_gather(
+            *(
+                rabbit_client.subscribe(
+                    exchange_name, functools.partial(parser_fct, app), **queue_kwargs
+                )
+                for exchange_name, parser_fct, queue_kwargs in EXCHANGE_TO_PARSER_CONFIG
             )
+        )
+
     yield
 
     # cleanup?
+    with log_context(_logger, logging.INFO, msg="Subscribing to rabbitmq channels"):
```

Comment:
you can also
```python
with log_context(...)
      rabbit_client: RabbitMQClient = get_rabbitmq_client(app)
      
     yield
     
     await logged_gather(...)
     
```

Reply:
```
>>> from contextlib import contextmanager
>>> @contextmanager
... def context():
...    print("start")
...    yield
...    print("stop")
... 
>>> def foo():
...   with context():
...      yield 1
...   yield 2
... 
>>> g = foo()
>>> next(g)
start
1
>>> next(g)
stop
2
>>> 
```

Reply:
ah sorry I forgot to adapt the message. done.